src/common/interfaces.ts

```diff
@@ -1,4 +1,4 @@
-import { BBox2d } from '@turf/helpers/dist/js/lib/geojson';
+import { BBox, BBox2d, MultiPolygon, Polygon } from '@turf/helpers/dist/js/lib/geojson';
```

Comment:
split import to import everything apart from bbox2d from @turf/turf

Reply:
fixed